### PR TITLE
Support legacy instant phase

### PR DESCRIPTION
**Pull request by sluongng**

### CODE REVIEW

File: pkg/io/data.go [hunk 1]
```diff
@@ -155,6 +157,31 @@ type jsonCounterEvent struct {
...
+		if err != nil {
+			return err
+		}
+
+		ce.Values[k] = f
+	}
+
+	return nil
+}
+
```
Comment (omaskery) [inline pkg/io/data.go]: I think this part is from your other pull request, could you remove it from this PR please? :slightly_smiling_face:

**Review by omaskery — CHANGES_REQUESTED** [review]
Thanks for your contribution!

The only requested change is that I think you've accidentally included some of your changes from PR #3. Once those have been removed from this PR this can be approved & merged :tada:

File: pkg/io/data.go (hunk 1 above)
Comment (sluongng) [reply pkg/io/data.go]: Sorry about that, I forgot I used master branch for this PR so I accidentally updated it

**Review by omaskery — APPROVED** [review]
Ace, thanks for your contribution! :) :tada:

**Pull request merged**